Resources/config/services.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="doctrine_migrations.diff_command" class="Doctrine\Bundle\MigrationsBundle\Command\MigrationsDiffDoctrineCommand">
```

Comment:
`public="false"` for all commands

Reply:
It will break BC because services are auto-registered as public services, no ?

Reply:
it should work with pre3.4 SFs

Reply:
NVM, it doesn't work before 3.1 while this bundle supports 2.3+. Too bad

Reply:
OK. Done!

Reply:
Should be reverted, my bad sorry.

Reply:
No problem. Hope I will not have to revert it again: smile: